bindings/sundials4py/test/test_arkstep.py

```diff
@@ -0,0 +1,124 @@
+#!/bin/python
+# -----------------------------------------------------------------
+# Programmer(s): Cody J. Balos @ LLNL
+# -----------------------------------------------------------------
+# SUNDIALS Copyright Start
+# Copyright (c) 2025-2025, Lawrence Livermore National Security,
+# University of Maryland Baltimore County, and the SUNDIALS contributors.
+# Copyright (c) 2013-2025, Lawrence Livermore National Security
+# and Southern Methodist University.
+# Copyright (c) 2002-2013, Lawrence Livermore National Security.
+# All rights reserved.
+#
+# See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# SUNDIALS Copyright End
+# -----------------------------------------------------------------
+
+import pytest
+import numpy as np
+from fixtures import *
+from sundials4py.core import *
+from sundials4py.arkode import *
+from problems import AnalyticODE, AnalyticMultiscaleODE
+
+
+@pytest.mark.skipif(
+    sunrealtype == np.float32, reason="Test not supported for sunrealtype=np.float32"
+)
+def test_explicit(sunctx):
+    y = N_VNew_Serial(1, sunctx)
+
+    ode_problem = AnalyticODE()
+
+    ode_problem.set_init_cond(y)
+
+    ark = ARKStepCreate(lambda t, y, ydot, _: ode_problem.f(t, y, ydot), None, 0, y, sunctx)
+
+    status = ARKodeSStolerances(ark.get(), SUNREALTYPE_RTOL, SUNREALTYPE_ATOL)
```

Comment:
I remember you saying something about being able to remove `.get()`, was that just for other objects (that are non-void*)?